sources/packages/web/src/components/partial-view/DesignationAgreement/DesignationAgreementSummary.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <content-group>
+    <empty-table-template
+      :isEmpty="!designations.length"
+      emptyMessage="You don’t have any agreements yet"
+    >
+      <template #image>
+        <v-img
+          height="200"
+          alt="You don’t have any agreements yet"
+          src="@/assets/images/designation_summary.svg"
+        />
+      </template>
+      <DataTable
```

Comment:
should this `<datatable>`  be inside `<empty-table-template></empty-table-template>`?

Reply:
It is actually inside _empty-table-template_ in the default slot. I am not sure it I am answering the question properly, please let me know.

Reply:
Component renamed to _toggle-content_ and adjusted. Please check and lets discuss as needed.